tools/devctr/Dockerfile

```diff
@@ -50,6 +50,10 @@ RUN apt-get update \
         tini \
         # for mdl
         ruby \
+        # for building kernel
+        flex \
+        bison \
+        bc \
```

Comment:
Building a kernel is a somewhat rare activity. Maybe we could install these tools just before building the kernel?

Reply:
Surely, building a kernel is relatively rare compared to running test codes / building firecracker. If devctr should be kept in minimal, I can move them just before building the kernel (same like rootfs build script does).

Reply:
Fixed.